src/screens/IndexScreen.js

```diff
@@ -149,7 +156,16 @@ export const IndexScreen = ({ navigation, route }) => {
   return (
     <SafeAreaViewFlex>
       {query === QUERY_TYPES.POINTS_OF_INTEREST ? (
-        <MapSwitchHeader setShowMap={setShowMap} showMap={showMap} />
+        <View>
+          <MapSwitchHeader setShowMap={setShowMap} showMap={showMap} />
+          <WrapperHorizontal>
+            <WrapperRow style={styles.textWrapper}>
+              <RegularText>{texts.pointOfInterest.filterByOpeningTime}</RegularText>
+              <Switch switchValue={filterByOpeningTimes} toggleSwitch={setFilterByOpeningTimes} />
+            </WrapperRow>
```

Comment:
i will check if there is a reason not to use it as in the waste calendar reminder.

i don't like putting an `ActivityIndicator` there, as we would need to somehow link it to the loading state of the query, and then it would be spinning for any initial loading as well. and arbitrary spinner shown for a fixed period also seems wrong to me. the switching is not the thing that is in a loading state here anyways.

do you have a better idea?

Reply:
if there is no usable loading state then we can not have one.